Backpack/build.gradle

```diff
@@ -102,10 +78,5 @@ task generateDokkaDocs(type: DokkaTask) {
   dependsOn("generateDokkaStatics")
 }
 
-task testOssDebugUnitTest {
-  dependsOn("testDebugUnitTest")
-}
-
-task lintOssDebug {
-  dependsOn("lintDebug")
-}
+apply from: "$rootProject.projectDir/android-test-configuration.gradle"
```

Comment:
how come these are separate files, rather than being part of `android-configuration`?

Reply:
Testing task with the same name is defined in app module by flavours – we get gradle error if we redefine it manually.